### ISSUE TITLE
Crashlytics does not start - waits for bug infinitely 

### ISSUE THREAD

**Issue by SzymonMatysik** [firebase/firebase-ios-sdk#10734]
### Description

Hi,

I've got an issue with crash reports. The Crashlytics successfully detected the app but it cannot receive any crash from my app. I've tried everything, also created another project - on the other project everything works fine.

At initialization the sample app where everything works fine displays:
```
2023-01-24 16:01:48.494101+0100 XXX[xxx:xxxxx] [Firebase/Crashlytics] Version 9.6.0
``` 

But on my main app it displays:
```
2023-01-24 16:14:00.553793+0100 XXX[xxx:xxxxx] 9.6.0 - [FirebaseCore][I-COR000001] Configuring the default app.
```

Moreover what I've noticed in console are lots of issues with duplicated firebase framework files:
```
objc[11887]: Class FIRCoreDiagnosticsLog is implemented in both /Users/XXX/Library/Developer/Xcode/DerivedData/XXX-XXXXXXXXXXXXXXXXXXXXX/Build/Products/Dev-iphonesimulator/PackageFrameworks/FirebaseCrashlytics_-F5B0BD00025CD43_PackageProduct.framework/FirebaseCrashlytics_-F5B0BD00025CD43_PackageProduct (0x112543070) and /Users/XXX/Library/Developer/CoreSimulator/Devices/XXXXXXXXXXXX/data/Containers/Bundle/Application/XXXXXXXXXXX/XXX.app/XXX (XXX). One of the two will be used. Which one is undefined.
```

I ran out of ideas, how to fix it, looking forward for help.

### Firebase SDK Version

9.6.0

### Xcode Version

14.2

### Installation Method

Swift Package Manager

### Firebase Product(s)

Crashlytics

### Targeted Platforms

iOS

**Comment by rizafran** [comment]
Thanks for reaching out, @SzymonMatysik. Regarding the duplicate Firebase framework warning _`One of the two will be used. Which one is undefined.`_, I was able to reproduce it when I installed the SDK on both SPM and CocoaPods. Could you check if you've installed the SDK as well on CocoaPods?
 
Also, could you verify if you're getting `Completed report submission` in your logs? You may try to enable your debug logging by following the steps [here](https://firebase.google.com/docs/crashlytics/test-implementation?platform=ios#enable-debug-logging).

**Comment by paulb777** [comment]
Those linker warnings should be addressed as they can cause undeterministic behavior in several ways.  There is possibly something in the Podfile causing pods to be doubly linked into the app and another dynamic framework.

**Comment by SzymonMatysik** [comment]
Thank you for responses!!! Your comments made me hop on right path to resolve the issue. 

First of all, I'll answer your questions:

@rizafran 
In my project I have only one dependency (sick!) on Cocoapods (well I should consider importing it as static lib and drop pods lol), which isn't connected with Google/Firebase services at all. To confirm that pods are not the issue, I've deintegrated them (one lib) and installed again. The same issue persisted. 
Second question: I didn't get any log from Crashlytics, nothing about "started running" nothing about "completed submission" just blank - On the other hand I did receive Analytics and other logs nice and clear.. This drove me nuts.

@paulb777 
YES! Exactly, but not the pods... by my mistake I have imported the SPM FirebaseCrashlytics which were imported to main target, to another module. Thats why Xcode sent me those warnings and the lib didn't start at all, which is confusing... 

because I got another warning mentioning different lib and the other lib works perfectly fine 🤔

**SOLUTION:**

> Deleting the FirebaseCrashlytics from other than the target in which were originally imported by SPM makes all should work smoothly fine! 

It's quite rare issue and made by human mistake, so I wish you all not to experience it by yourself and keep in mind my findings 🥲 
Cheers
